File: framework/languages/dotnet/samples/ShoppingMall/Server/Configuration/SampleFlowLog.cs
namespace ShoppingMall.Server.Configuration;

public static class SampleFlowLog
{
    public static string Path(string role)
    {
        var dir = Environment.GetEnvironmentVariable("SHOPPINGMALL_LOG_DIR") ?? "logs";
        return System.IO.Path.Combine(dir, $"flow-{role}.log");
    }
}

File: framework/languages/java/samples/java/ShoppingMall/Server/Configuration/src/main/java/systems/zlink/samples/shoppingmall/server/configuration/SampleFlowLog.java
package systems.zlink.samples.shoppingmall.server.configuration;

public final class SampleFlowLog {
    private SampleFlowLog() {
    }

    public static String path(String role) {
        return System.getenv().getOrDefault("SHOPPINGMALL_LOG_DIR", "logs")
            + "/flow-" + role + ".log";
    }
}
